src/basic/errno-util.h

```diff
@@ -94,3 +94,10 @@ static inline bool ERRNO_IS_NOT_SUPPORTED(int r) {
                       ENOTTY,
                       ENOSYS);
 }
+
+/* Two different errors for access problems */
+static inline bool ERRNO_IS_PRIVILEGE(int r) {
```

Comment:
PRIVILEGED? Maybe call this ERRNO_IS_PERMISSIONS. I think it'd be nicer to squash this commit with the one that makes use of the macro too.

Reply:
my thinking was that that the error is a "privilege error" hence the question "errno is privilege?", i.e. a "privileged error" would mean the fact that the error happened is secret, while a "privilege error" means that you didn't have the privilege to do something.

And I wanted to avoid the word "PERMISSIONS" since otherwise one might think this is just about EPERM... There's enough confusion around EACCES and EPERM (since EACCES's strerror() string uses the word "permission"), hence I wanted a different word that could cover both.

(The reason this is a separate commit is mostly that it is also included in the pidfd_open PR and in my homed PR this way ;-))